blocks/library/embed/index.js

```diff
@@ -458,5 +470,11 @@ registerBlockType(
 	getEmbedBlockSettings( {
 		title: 'YouTube',
 		icon: 'video-alt3',
+		name: 'core-embed/youtube',
```

Comment:
Fine either way tbh. Still thinking maybe it makes more sense to pass attributes rather than a block, and create the block in the patterns plugin. That should always be the same anyway. I think I did the same for paste.